sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Blobs/src/Microsoft.Azure.WebJobs.Extensions.Storage.Blobs.csproj

```diff
@@ -17,6 +17,12 @@
     <Compile Include="$(MicrosoftAzureWebJobsExtensionsClientsSources)\**\*.cs" Link="Shared\%(RecursiveDir)\%(Filename)%(Extension)" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Compile Include="..\..\..\eventgrid\Microsoft.Azure.WebJobs.Extensions.EventGrid\src\TriggerBinding\HttpRequestProcessor.cs" Link="EventGrid\HttpRequestProcessor.cs" />
```

Comment:
This might become a problem. We don't run storage tests when something in the `eventgrid` directory changes. So it's easy to unintentionally break something.

Reply:
What other options do we have here? 
I see that this approach is used in other places:
https://github.com/Azure/azure-sdk-for-net/blob/7705656e645e5bc1e9afb07fe33f65a5a565da63/sdk/storage/Azure.Storage.Blobs/src/Azure.Storage.Blobs.csproj#L27

Eventually we will run storage test begore a release.